Automattic/pocket-casts-ios
# Discover Collection View: Feature Flag

| 📘 Part of: #2104 |
|:---:|

* Adds the `discoverCollectionView` Feature Flag
* Adds a `DiscoverCollectionViewController` view controller with a single 

## To test

* Enable the `discoverCollectionView` feature flag
* Restart the app - I might make this dynamic later but this is just for the flag
* Navigate to the "Discover" tab
* Verify that a "Discover" label is shown - this is a temporary placeholder

## Checklist
